src/Native/System.Security.Cryptography.Native/pal_ssl.cpp

```diff
@@ -66,7 +66,16 @@ extern "C" const SSL_METHOD* CryptoNative_TlsV1_2Method()
 
 extern "C" SSL_CTX* CryptoNative_SslCtxCreate(SSL_METHOD* method)
 {
-    return SSL_CTX_new(method);
+    SSL_CTX* ctx = SSL_CTX_new(method);
+#ifdef SSL_OP_NO_COMPRESSION
```

Comment:
I believe it was added after 0.9.8a, so if you used a version prior to that.  I think it's likely that other aspects of the code would fail to compile if you tried to use such a version, so I could remove these if we consider it a problem, but I figured I'd put it in for good measure.